aptos-move/move-examples/token_objects/sources/token.move

```diff
@@ -53,24 +50,29 @@ module token_objects::token {
         mutation_events: event::EventHandle<MutationEvent>,
     }
 
+    /// This enables burning an NFT, if possible, it will also delete the object. Note, the data
+    /// in inner and self occupies 32-bytes each, rather than have both, this data structure makes
+    /// a small optimization to support either and take a fixed amount of 34-bytes.
+    struct BurnRef has drop, store {
```

Comment:
Can we be more consistent with refs vs caps?

Reply:
I've called them refs throughout my code. Per @wrwg , a capability has certain properties that these do not have. That is they are revocable. A ref could live within a capability, but it is not a capability itself.